<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="config" default="$(find path_tracking)/config/config.yaml" />
  <node pkg="path_tracking" type="python_sim.py" name="python_sim" output="screen">
    <rosparam command="load" file="$(arg config)"/>
  </node>
</launch>
